### ISSUE TITLE
Lua errors on format string

### ISSUE THREAD

**Issue by legracen** [AIE-Guild/GreenWall#139]
Error pops up continually.

```
Message: Interface\AddOns\GreenWall\Utility.lua:100: bad argument #3 to 'format' (string expected, got no value)
Time: Sat Aug 11 08:08:26 2018
Count: 4
Stack: Interface\AddOns\GreenWall\Utility.lua:100: bad argument #3 to 'format' (string expected, got no value)
[C]: ?
[C]: in function `format'
Interface\AddOns\GreenWall\Utility.lua:100: in function `Debug'
Interface\AddOns\GreenWall\Chat.lua:115: in function `ReplicateMessage'
Interface\AddOns\GreenWall\GreenWall.lua:320: in function `GreenWall_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

Locals: 
```

**Comment by mrogaski** [comment]
Reported by Deltethnia:
```
Message: Interface\AddOns\GreenWall\Utility.lua:100: bad argument #3 to 'format' (string expected, got no value)
Time: Fri Aug 10 23:48:07 2018
Count: 1
Stack: Interface\AddOns\GreenWall\Utility.lua:100: bad argument #3 to 'format' (string expected, got no value)
Interface\AddOns\GreenWall\GreenWall.lua:320: in function `GreenWall_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

Locals: self = GreenWallInterfaceFrame {
0 = <userdata>
cancel = <function> defined @Interface\AddOns\GreenWall\GreenWall.lua:196
name = "GreenWall"
refresh = <function> defined @Interface\AddOns\GreenWall\GreenWall.lua:194
okay = <function> defined @Interface\AddOns\GreenWall\GreenWall.lua:195
default = <function> defined @Interface\AddOns\GreenWall\GreenWall.lua:197
}
event = "CHAT_MSG_CHANNEL_LEAVE"
_ = ""
player = "Maryprankstr-ThoriumBrotherhood"
_ = ""
_ = "1. tec"
_ = ""
_ = ""
_ = 0
number = 1
semver = <table> {
__lt = <function> defined @Interface\AddOns\GreenWall\Lib\SemanticVersion.lua:167
__eq = <function> defined @Interface\AddOns\GreenWall\Lib\SemanticVersion.lua:163
__le = <function> defined @Interface\AddOns\GreenWall\Lib\SemanticVersion.lua:171
__tostring = <function> defined @Interface\AddOns\GreenWall\Lib\SemanticVersion.lua:97
__index = <table> {
}
new = <function> defined @Interface\AddOns\GreenWall\Lib\SemanticVersion.lua:50
}
```

**Comment by NoobTaco** [comment]
Getting the same error

`17x GreenWall\Utility.lua:100: bad argument #3 to 'format' (string expected, got no value)
[C]: in function `format'
GreenWall\Utility.lua:100: in function `Debug'
GreenWall\Chat-Chat.lua:115: in function `ReplicateMessage'
GreenWall\GreenWall-1.9.6.lua:320: in function `GreenWall_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

Locals:
(*temporary) = "sender=%s, language=%s, target=%s, flags=%s, line=%s, guid=%s"
(*temporary) = ""
(*temporary) = "string expected, got no value"`

**Comment by mrogaski** [comment]
Fixed in #140
